.npmignore

```diff
@@ -3,3 +3,4 @@ __tests__/
 coverage/
 .eslint*
 .prettier*
+jest.*.js
```

Comment:
Should we add this in `rdme` too?

Reply:
Yeah might as well

Reply:
added here: https://github.com/readmeio/rdme/commit/243c8b02c2e7e258bbb067bf7c8eb7c321e941d7